bootstrap/api/responses.go

```diff
@@ -67,14 +67,16 @@ type channelRes struct {
 }
 
 type viewRes struct {
-	MFThing     string          `json:"mainflux_id,omitempty"`
-	MFKey       string          `json:"mainflux_key,omitempty"`
+	ThingID     string          `json:"thing_id,omitempty"`
+	ThingKey    string          `json:"thing_key,omitempty"`
 	Channels    []channelRes    `json:"mainflux_channels,omitempty"`
 	ExternalID  string          `json:"external_id"`
```

Comment:
if `mainflux_id` can be changed to `thing_id`,  `mainflux_key` to `thing_key` , then `mainflux_channels` can be changed to `channels`

Reply:
@arvindh123 theres a clash between requests and responses, what do you suggest https://github.com/SammyOina/mainflux/blob/70f53c2f979dbe408d9b1ad7ba0ba99a72eb0c70/pkg/sdk/go/bootstrap.go#L30-L35